datavault-webapp/src/main/webapp/WEB-INF/freemarker/admin/vaults/index.ftl

```diff
@@ -18,51 +39,121 @@
+                                <a href="${springMacroRequestContext.getContextPath()}/vaults/${vault.getID()}">${vault.name?html}</a>
                             </td>
-                            <td>${vault.name?html}</td>
-                            <td>${vault.getUserName()?html} (${vault.getUserID()?html})</td>
-                            <td>${vault.getSizeStr()}</td>
-                            <td>${vault.policyID?html}</td>
+                            <td>${vault.getNumberOfDeposits()}</td>
+                           <td>${vault.getSizeStr()}</td>
+                            <td>
+                            <a href="${springMacroRequestContext.getContextPath()}/vaults/${vault.getID()}/${vault.getUserID()}">${vault.getUserName()?html} (${vault.getUserID()?html})</a>
+                            </td>                           
+                            
                             <td>${vault.groupID?html}</td>
-                            <td>${vault.getCreationTime()?datetime}</td>
                             <td>${vault.reviewDate?string('dd/MM/yyyy')}</td>
+                            <td>${vault.getCreationTime()?datetime}</td>
+                            <td style="display:none">
+                            <!-- TODO : Hiding the remove button unless the functionality is clear as to what it should do - Soft/Hard delete -->
+                            <a class="btn btn-xs btn-danger pull-left" href="#" data-vault="${vault.ID}" data-toggle="modal" data-target="#confirm-removal">
+                                <span class="glyphicon glyphicon-remove" aria-hidden="true"></span> Remove
+                          </a>
+                            </td>
                         </tr>
                     </#list>
                 </tbody>
             </table>
         </div>
-
-        <form id="search-vaults" class="form" role="form" action="${springMacroRequestContext.getContextPath()}/admin/vaults/csv" method="get">
-            <div class="input-group">
-                <input type="text" class="form-control hidden" value="${query}" name="query" placeholder="Search for...">
-                <button class="btn btn-primary" type="submit"><span class="glyphicon glyphicon-export" aria-hidden="true"></span> Export to CSV</button>
+            
+            <div align="center">
+            	<p class="text-info">${recordsInfo}</p>
             </div>
-        </form>
+            <div align="center">
+            <nav aria-label="...">
+			    <ul class="pagination pagination-lg" id="paginationButton">
+            		<#list pages as page>
+				    	<li class="page-item" id="${page}">
+				     	 <a class="page-link" href="${springMacroRequestContext.getContextPath()}/admin/vaults?sort=${theSort}&order=${theOrder}&pageId=${page}" tabindex="-1">${page}</a>
+				    	</li>
+					</#list>
+				 </ul>
+			</nav>
+        </div>
+
     </#if>
 
 </div>
+ 
+<script>
+
+$(function(){
```

Comment:
Not sure I understand what that function does exactly but it looks like some of it is not necessary?

Reply:
This function will make the pagination button active  whenever it is clicked. By default the first page is made active else the depending on the href and the url in the browser the particular pageId button is made to look active and rest of the buttons looks inactive

Reply:
Yea I can understand that would be useful if we were using Ajax but I think it's reloading the page? So can't we figure out which button is active using the parameters instead of using javascript?

Reply:
We can identify the page id and pass a flag to check which one is active and add the class to make it active. I don't understand what will be the problem if we use Javascript.

Reply:
Removed the javascript function and updated the code by adding the flag to check which page is active.Changes can be reviewed in below commit
https://github.com/DataVault/datavault/commit/74bc4ea62fa324401555c59405dda1f887f6b2e5